catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/openlayers.polygon.js

```diff
@@ -49,13 +49,15 @@ define([
 
                 _.each(coords, function(item) {
                     _.each(item, function(point) {
+                        point = ol.proj.transform([point[0], point[1]], properties.projection, 'EPSG:4326');
+
```

Comment:
✏️  Extra newline here but not in the other

Reply:
:/   Fixed